e2e/nomostest/clean.go

```diff
@@ -271,16 +271,24 @@ func isTestLabel(key string) bool {
 func deleteConfigSyncAndTestAnnotationsAndLabels(nt *NT, ns *corev1.Namespace) error {
 	var annotations = make(map[string]string)
 	var labels = make(map[string]string)
+	changed := false
```

Comment:
nit: This might be cleared as `found`, since it's looking for CS labels and annotations. We don't actually know if it was changed and we're not really checking for other changes.

Reply:
Agreed that seems more clear